databricks_sql_connector_core/src/databricks_sql_connector_core/sql/client.py

```diff
@@ -1196,7 +1220,44 @@ def fetchmany_arrow(self, size: int) -> pyarrow.Table:
 
         return results
 
-    def fetchall_arrow(self) -> pyarrow.Table:
+    def merge_columnar(self, result1, result2):
```

Comment:
can you move the code change into a separated PR? Let this PR only focus the repo restructure.